content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,256 @@
+
+The first time period column represents the current time period. For example, if the |MPS| is
+configured to use months, the first column displays data for the current month. On this first
+column, the :guilabel:`+ Suggested Replenishment` field appears in one of five colors:
+
+- :guilabel:`Green` means that a replenishment order must be generated to keep stock at the
+  :guilabel:`Safety Stock Target`.
+- :guilabel:`Gray` means that a replenishment order has already been generated to keep stock at the
+  :guilabel:`Safety Stock Target`.
+- :guilabel:`Yellow` means that a replenishment order has already been generated, but the quantity
+  it was created for is not enough to keep stock at the :guilabel:`Safety Stock Target`.
+- :guilabel:`Red` means that a replenishment order has already been generated, but the quantity it
+  was created for puts the amount of stock above the :guilabel:`Safety Stock Target`.
+
+The :guilabel:`+ Suggested Replenishment` field appears white if no replenishment order has been
+generated, and it is not necessary to generate one at the current moment.
+
+Add a product
+=============
 
-.. tip::
-        In the MPS settings, you can define the time range of your MPS (month/week/day)
-        and the number of periods you want to display at all times.
+To use |MPS| to manage the replenishment of a product, navigate to :menuselection:`Manufacturing app
+--> Planning --> Master Production Schedule`. At the top of the |MPS| page, click :guilabel:`Add a
+Product` to open the :guilabel:`Add a Product` pop-up window.
 
-Now, go to :menuselection:`Planning --> Master Production Schedule` and click on *add a
-product*. You can now define your safety stock target (= the stock you
-want to have on hand at the end of the period) and the minimum and
-maximum quantities that must or can be replenished in each period.
+.. important::
+   Products must be properly configured to be replenished through |MPS|.
 
-.. image:: use_mps/mps_1.png
-    :align: center
+   In the case of manufactured products, the :guilabel:`Manufacture` route must be selected in the
+   :guilabel:`Routes` section of the :guilabel:`Inventory` tab, on the product's page.
 
-In the MPS view, you can decide which information you would like to display
-by clicking on *rows*. For instance, the *Actual demand* will show you which
-quantity of products has already been ordered for the period, or *Available to Promise*,
-what can still be sold during that same period (what you plan to replenish - what
-is already sold during the period). You can also decide to hide rows if you like.
+   In the case of products that are purchased, the :guilabel:`Buy` route must be selected in the
+   :guilabel:`Routes` section of the :guilabel:`Inventory` tab, on the product's page. Additionally,
+   a vendor and the price they sell the product for must also be specified on the
+   :guilabel:`Purchase` tab.
```

Comment:
in this section, since you're not explicitly giving instructions to configure the routes of the products, I think italicizing all the items that you've used guilabels for would be the move here. Also, it might be worthwhile to link readers to the relevant documentation pages where specific details on how to configure these are written.